symfony/plugins/orangehrmAdminPlugin/Service/LicenseService.php

```diff
@@ -17,84 +17,116 @@
  * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  * Boston, MA  02110-1301, USA
  */
-class LicenseService extends BaseService {
-    
-    private $licenseDao;
-    
+
+namespace OrangeHRM\Admin\Service;
+
+use OrangeHRM\Admin\Dao\LicenseDao;
+use OrangeHRM\Admin\Dto\LicenseSearchFilterParams;
+use OrangeHRM\Core\Exception\DaoException;
+use OrangeHRM\Entity\License;
+
+class LicenseService
+{
     /**
-     * @ignore
+     * @var LicenseDao|null
+     *
      */
-    public function getLicenseDao() {
-        
+    private ?LicenseDao $licenseDao = null;
+
+    /**
+     * @return LicenseDao
+     */
+    public function getLicenseDao(): LicenseDao
+    {
         if (!($this->licenseDao instanceof LicenseDao)) {
             $this->licenseDao = new LicenseDao();
         }
-        
+
         return $this->licenseDao;
     }
 
     /**
-     * @ignore
+     * @param $licenseDao
+     * @return void
      */
-    public function setLicenseDao($licenseDao) {
+    public function setLicenseDao($licenseDao)
+    {
         $this->licenseDao = $licenseDao;
     }
-    
+
     /**
      * Saves a license
-     * 
+     *
      * Can be used for a new record or updating.
-     * 
-     * @version 2.6.12 
-     * @param License $license 
+     *
+     * @param License $license
      * @return NULL Doesn't return a value
+     * @throws \DaoException
+     * @version 2.6.12
      */
-    public function saveLicense(License $license) {        
-        $this->getLicenseDao()->saveLicense($license);        
+    public function saveLicense(License $license)
```

Comment:
return type. may be you can return `$this->getLicenseDao()->saveLicense($license);`